nautobot/docs/release-notes/version-1.0.md

```diff
@@ -191,6 +191,7 @@ The "Related Devices" table has been removed from the detailed Device view.
 - [#417](https://github.com/nautobot/nautobot/pull/417) - Fixed incorrect link to Docker docs from installation docs
 - [#428](https://github.com/nautobot/nautobot/issues/428) - Fixed GraphQL error when handling ASNs greater than 2147483647
 - [#430](https://github.com/nautobot/nautobot/pull/430) - Fixed missing `ContentType` foreign keys in GraphQL
+- [#448](https://github.com/nautobot/nautobot/issues/448) - Fixed nautobot-server init overwriting configuration
```

Comment:
I'd consider this to be an item for `Changed` personally - "`nautobot-server init` no longer provides an option to overwrite existing config files"

Reply:
I am good with this, just updated.